Mail/MailApp.swift

```diff
@@ -67,6 +67,8 @@ struct MailApp: App {
                 .onChange(of: scenePhase) { newScenePhase in
                     switch newScenePhase {
                     case .active:
+                        @LazyInjectService var appLaunchCounter: AppLaunchCounter
```

Comment:
As a rule of thumb, it is better to use the DI this way:

- `LazyInjectService` as an ivar (like the account manager) _if possible_ so the resolution is done once.
If not possible to use an ivar, fallback with:
- `InjectService` used directly inside the scope of a function, the lazy will not be usefull here.

Here I think you can move this `@LazyInjectService var appLaunchCounter: AppLaunchCounter` just under `@LazyInjectService private var accountManager: AccountManager`